client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/index.tsx

```diff
@@ -19,9 +22,21 @@ const Launchpad: Step = ( { navigation }: LaunchpadProps ) => {
 	const translate = useTranslate();
 	const almostReadyToLaunchText = translate( 'Almost ready to launch' );
 	const siteSlug = useSiteSlugParam();
-
+	const emailedVerified = useVerifiedParam();
 	const site = useSite();
 	const launchpadScreenOption = site?.options?.launchpad_screen;
+	const dispatch = useDispatch();
+
+	useEffect( () => {
+		if ( emailedVerified ) {
+			const message = translate( 'Email confirmed!' );
+			dispatch(
+				successNotice( message, {
+					duration: 10000,
+				} )
+			);
+		}
+	}, [] );
```

Comment:
I think we can add the param in the dependency array just in case (althought it seems to work here usually not having the variable in the dependency array will result in a stale closure)

Reply:
I added an empty array because I only want this useEffect logic to trigger on the first render. We only need to see this `Email confirmed` notification one time when the launchpad page initially renders based on the 'verified' query param.